src/pages/workspaces/workspace/modals/FindWorkflowModal.tsx

```diff
@@ -0,0 +1,142 @@
+import { icon, Link, Modal } from '@terra-ui-packages/components';
+import _ from 'lodash/fp';
+import React from 'react';
+import { getEnabledBrand } from 'src/libs/brand-utils';
+import colors from 'src/libs/colors';
+import { getConfig } from 'src/libs/config';
+import { isFeaturePreviewEnabled } from 'src/libs/feature-previews';
+import { FIRECLOUD_UI_MIGRATION } from 'src/libs/feature-previews-config';
+import * as Nav from 'src/libs/nav';
+import * as Utils from 'src/libs/utils';
+
+interface WorkflowSourceCardProps {
+  title: string;
+  description: string;
+  url: string;
+  openInNewTab: boolean;
+}
+
+const WorkflowSourceCard = (props: WorkflowSourceCardProps) => {
+  return (
+    <div
+      style={{
+        backgroundColor: colors.accent(0.1),
+        padding: '0.8em',
+        borderRadius: '8px',
+        lineHeight: '18px',
+      }}
+    >
+      <div style={{ marginLeft: '10px' }}>
+        <h4>
+          <Link href={props.url} {...(props.openInNewTab ? Utils.newTabLinkProps : {})}>
+            {props.title}
+            {props.openInNewTab
```

Comment:
This might be clearer if we use `&&` instead of the ternary operator here.

Reply:
Can you explain how to use `&&` inside the element tag? 🤔

Reply:
Sorry, the placement of my comment was a bit confusing. I meant to suggest trying `{props.openInNewTab && icon(...)}` instead of `{props.openInNewTab ? icon(...) : ''}` on line 33.

Reply:
Ah I see. Yeah that is definitely more clearer. I have updated it to use `&&`. Thanks!